src/js/components/header/mobile.js

```diff
@@ -1,8 +1,25 @@
-var $ = require('jquery');
+var select = require('../../utils/select');
+var addClass = require('../../utils/add-class');
+var removeClass = require('../../utils/remove-class');
+var dispatch = require('../../utils/dispatch');
 
-$('.usa-menu-btn, .usa-overlay, .usa-nav-close').on('click touchstart', function (e) {
-  $('.usa-overlay, .usa-nav').toggleClass('is-visible');
-  $('body').toggleClass('usa-mobile_nav-active');
-  $('.usa-nav-close').focus();
-  e.preventDefault();
+var navElements = select('.usa-menu-btn, .usa-overlay, .usa-nav-close');
+var toggleElements = select('.usa-overlay, .usa-nav');
+var navCloseElement = select('.usa-nav-close')[ 0 ];
+
+navElements.forEach(function (element) {
+  dispatch(element, 'click touchstart', function (e) {
+    toggleElements.forEach(function (element) {
+      toggleClass(element, 'is-visible');
+    });
+    toggleClass(document.body, 'usa-mobile_nav-active');
+    navCloseElement.focus();
+    return false;
+  });
 });
+
+function toggleClass (element, className) {
```

Comment:
My attitude so far has been to leave potential util functions in their existing files if they're only used in that place. Once more than one component needs them, I pull them out (like `addClass` & `removeClass` elsewhere in this PR).